myapp/src/Day2/Que11/CartContext.jsx

```diff
@@ -0,0 +1,28 @@
+// CartContext.js
+import React, { createContext, useContext, useState } from 'react';
+
+const CartContext = createContext();
+
+export function CartProvider({ children }) {
+  const [cart, setCart] = useState([]);
+
+  // Add item to cart
+  const addToCart = (product) => {
+    setCart((prevCart) => [...prevCart, product]);
+  };
+
+  // Remove item from cart
+  const removeFromCart = (productId) => {
+    setCart((prevCart) => prevCart.filter((item) => item.id !== productId));
+  };
+
+  return (
+    <CartContext.Provider value={{ cart, addToCart, removeFromCart }}>
+      {children}
+    </CartContext.Provider>
+  );
+}
+
+export function useCart() {
```

Comment:
@anantmishra-successive this is not required. 'use' prefix is generally reserved for custom hooks. For dealing with context, this is redundant.